accord-core/src/main/java/accord/local/Bootstrap.java

```diff
@@ -232,8 +233,10 @@ private void started(SafeToRead state, @Nullable Timestamp maxApplied)
                         state.startedAt = logicalClock++;
                 }
                 // TODO (expected): associate callbacks with this CommandStore, to remove synchronization
-                FetchMaxConflict.fetchMaxConflict(node, state.ranges)
-                                .begin((executeAt, failure) -> safeToReadCallback(state, executeAt, failure));
+                FetchMaxConflict.fetchMaxConflict(node, store, state.ranges)
+                                .begin((executeAt, failure) -> {
+                                    store.execute(() -> safeToReadCallback(state, executeAt, failure));
```

Comment:
I think this implies that most of the time we are in the store, but there is an edge case that required you to pass it through?  There is a new method now to run right away when in-store else enqueue; `maybeExecuteImmediately`

Reply:
Updated to use `maybeExecuteImmediately`